docs/source/basic/cheatsheet.rst

```diff
@@ -115,9 +115,10 @@ Create a CPU device for memory allocation on the host side
 
 Allocate a buffer in host memory
   .. code-block:: c++
-
-     Vec<Dim, Idx> extent = value;
-     using BufHost = Buf<DevHost, DataType, Dim, Idx>;
+     // Use alpaka vector as a static array for the extent(dimensions)
+     alpaka::Vec<Dim, Idx> extent = value;
+     // Allocate memory for the alpaka buffer which is a dynamic array
+     using BufHost = alpaka::Buf<DevHost, DataType, Dim, Idx>;
```

Comment:
Be careful to keep the cheatsheet short! It is rather not an introduction, but really a cheatsheet for the API. At least that's my understanding.

Reply:
Sure, these are really important explanations. I am not planning to add more.